docker/smqtk_wrapper_python/main.py

```diff
@@ -0,0 +1,178 @@
+
+import flask
+import json
+import os
+import os.path
+import requests
+import re
+import sys
+import urllib.parse
+
+from hashlib import sha1
+from threading import Thread
+
+from content_type_map import extension_to_type
+
+RE_HTTP = re.compile(r'''^https?\://.*''')
+
+def log(s):
+    print(s)
+    sys.stdout.flush()
+
+def proxy(new_location):
+    log('  PROXYING TO %s' % new_location)
+    error_message = None
+
+    try:
+        proxy_res = requests.get(new_location, timeout=(30.05, 30.05))
```

Comment:
This choice was made on the advice of the [requests documentation](http://docs.python-requests.org/en/master/user/advanced/#timeouts) regarding timeouts.

 > The connect timeout is the number of seconds Requests will wait for your client to establish a connection to a remote machine (corresponding to the connect()) call on the socket. It's a good practice to set connect timeouts to slightly larger than a multiple of 3, which is the default TCP packet retransmission window.